File: labs/lab01/AquecimentoInterativo.java
package labs.lab01;

import java.util.Scanner;

public class AquecimentoInterativo {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String string;
        while(!(string = scanner.nextLine()).equals("FIM")){
            int num = contar(string);
            System.out.println(num);
            
        }
        scanner.close();
    }

    public static int contar(String string) {
        int tam = string.length();
        int cont = 0;
        for (int i = 0; i < tam; i++) {
            if (string.charAt(i) >= 'A' && string.charAt(i) <= 'Z') {
                cont++;
            }
        }
        return cont;
    }
}

File: labs/lab01/AquecimentoInterativo.c
#include <stdio.h>
#include <string.h>

int contar(char string[]) {
    int tam = strlen(string);
    int cont = 0;
    for(int i = 0; i < tam; i++) {
        if(string[i] >= 'A' && string[i] <= 'Z') {
            cont++;
        }
    }
    return cont;
}

int main() {

    char string[1000];
    while(1){
        scanf(" %[^\n]", string);

        if (strcmp(string, "FIM") == 0) {
            break;
        } else {
            char string[50];
        
            fgets(string, 50, stdin);
            string[strcspn(string, "\n")] = 0;

            int x = contar(string);
            printf("%d\n", x);
        }
    }
    return 0;
}
